src/robusta/runner/main.py

```diff
@@ -45,6 +45,8 @@ def main():
         logging.info(f"Running alerts workers pool of {ALERT_BUILDER_WORKERS}")
 
     Web.init(event_handler, loader)
+
+    event_handler.set_cluster_active(True)
```

Comment:
I think it's a redundant db update.
When the server starts, the RobustaSink updates ClusterStatus. ([here](https://github.com/robusta-dev/robusta/blob/acfadcbe8270474c2bac741d0be69403fc7a9720/src/robusta/core/sinks/robusta/dal/supabase_dal.py#L416))
We can just add there `active: true` and remove this call (which can also fail and interfere with server startup)

Reply:
That's correct, but from what Avi proposed, I made it so any sink can override this method and process being active as they want
So It resulted in two requests